.github/workflows/generate-installer.yml

```diff
@@ -20,41 +20,41 @@ on:
         description: "File Type"
         required: true
         options:
-          - iso
-          - dmg
-          - zip
```

Comment:
Will the workflows still run as it should?
The options no longer matches the if statements below